README.md

```diff
@@ -172,6 +173,12 @@ with open(path/to/model_file.pickle.dat, 'rb') as model_file:
     model = pickle.load(model_file)
 ```
 
+### Usage for the explain module
+The explain module takes a model, __which is not specifically tied to a CAPICE version__, and exports the feature importances of said model.
```

Comment:
I've tried running the explain tool an the original CAPICE model used in the publication:
```
python.exe capice.py explain -i xgb_booster.pickle.dat
Traceback (most recent call last):
  File "capice\src\molgenis\capice\capice.py", line 16, in <module>
    main()
  File "capice\src\molgenis\capice\capice.py", line 12, in main
    argument_handler.handle()
  File "capice\src\molgenis\capice\core\args_handler.py", line 34, in handle
    args.func(args)
  File "capice\src\molgenis\capice\cli\args_handler_explain.py", line 86, in _handle_args
    self._handle_module_specific_args(input_path, output_path, output_filename, args)
  File "capice\src\molgenis\capice\cli\args_handler_explain.py", line 91, in _handle_module_specific_args
    model = pickle.load(model_file)
_pickle.UnpicklingError: invalid load key, '\x0a'.
```
After reading the docs I thought this would work.